### DISCUSSION TITLE
how to run vscode development env

### DISCUSSION THREAD

**Discussion by ashwin31** [gitpod-io/openvscode-server#311] [Q&A] [answered]
I want to run vscode in development mode and try new features or customisations. 

Now I am running the following two commands in two terminals.
```
yarn watch
./scripts/code-web.sh 
```
is there any other way?

**Comment by filiptronicek** [comment, marked as answer]
Hey there @ashwin31 👋! Please feel free to look over our https://github.com/gitpod-io/openvscode-server/blob/main/.gitpod.yml to see what we execute in every terminal in our dev env 😄.

**Reply by ashwin31** [reply]
Thank you @filiptronicek
